plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/InMemoryThriftMetastore.java

```diff
@@ -205,7 +205,10 @@ public synchronized void createTable(HiveIdentity identity, Table table)
         }
 
         PrincipalPrivilegeSet privileges = table.getPrivileges();
-        if (privileges != null) {
+        if (privileges != null &&
```

Comment:
Is `privileges != null` same case where privileges are empty? I'm not sure these are semantically the same (e.g. maybe it's possible to ask for empty privileges)

Reply:
what's the resolution here? Did you drop the change?

Reply:
added now